charts/nango/templates/runner/runner-deployment.yaml

```diff
@@ -15,8 +15,9 @@ spec:
     spec:
       containers:
         - name: {{ .Values.runner.name | default "nango-runner" }}
-          image: nangohq/nango-runner:{{ .Values.runner.tag | default "enterprise" }}
+          image: nangohq/nango:prod-{{ .Values.runner.tag | default .Values.shared.tag }}
```

Comment:
Same

Reply:
600 I think but still valid point. I have script somewhere which I used at some point to do env vars update on all the runners